ADDING_NEW_RULES.md

```diff
@@ -0,0 +1,121 @@
+# Adding new rules
+
+We will want to add new rules to this validator as the [GTFS spec](https://github.com/google/transit/tree/master/gtfs) and the surrounding applications and tools change.  This page outlines the process of adding new rules to this tool.
+
+### 0. Prepare for implementation 
+* Check the list of [currently implemented rules](RULES.md) to make sure the rule doesn't already exist.
+* Check the list of [planned future rules](https://github.com/MobilityData/gtfs-validator/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+rule%22) to see if an issue already exists for the proposed rule.
+    * If no existing issue exists, open a new issue with the ["new rule" label](https://github.com/MobilityData/gtfs-validator/issues?q=is%3Aissue+is%3Aopen+label%3A%22new+rule%22).
+* Discuss the rule with the community via the Github issue and come to a general consensus on the exact logic, and if it should be an `ERROR` or `WARNING`.  Generally, errors are behavior that directly violate the GTFS documentation.  Warnings are behavior that is not advised (e.g., against best practices) but not explicitly forbidden in the GTSF documentation.
+* Implement new rule using the process below
+
+For the below example, let's look at implementing a new rule that [TODO]
+
+If you want to take a look at a complete set of changes that implement this new rule before diving into the instructions, see [TODO this commit on Github](https://github.com/MobilityData/gtfs-validator).
+
+### 1. Add the rule to [`RULES.md`](RULES.md)
+
+Add the rule to the error or warnings table at the top of [`RULES.md`](RULES.md):
+
+~~~
+
+~~~
+
+...and add a definition of that rule at the bottom of the errors or warnings section:
+
+~~~
+
+~~~
+
+### 2. Add a new `*Notice.java` class for the new rule output
+
+All classes that implements rules output are under the [`notice` package](https://github.com/MobilityData/gtfs-validator/tree/master/domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice) of the domain layer.
+They must extend either `ErrorNotice` or `WarningNotice` and fit the `*Notice` format.
+
+- Add the error code definition to either `ErrorNotice` or `WarningNotice` class located in the [`base`](https://github.com/MobilityData/gtfs-validator/tree/master/domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/base) package, for example
+
+`protected static final String E_013 = 13;`
+
+- Add your new class in the [`error`](https://github.com/MobilityData/gtfs-validator/tree/master/domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/error) or [`warning`](https://github.com/MobilityData/gtfs-validator/tree/master/domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/notice/warning) package accordingly
+
+- override and define an `export` method like the following.
```

Comment:
I would capitolize the first letter of all itemized steps for consistency, like `Override...`. It makes it easier to read.